db/backup/cleanup.yml

```diff
@@ -1,5 +1,4 @@
----
 tasks:
     db:backup:cleanup:
```

Comment:
Please update this line & the following lines to follow a 2-space indentation pattern. The YAML spec is not opinionated about 2 or 4 spaces, but IntelliJ products default to 2 spaces, which I certainly prefer.